bundles/net.tourbook/src/net/tourbook/map25/layer/marker/MarkerShape.java

```diff
@@ -39,4 +41,9 @@ public enum MarkerShape {
     */
    ARROW,
 
+   /**
+    * using compass for a compass rose
+    */
+   COMPASS,

```

Comment:
The compassrose should be similar like in the 3D map

1. Position should be in the top right corner
2. It should not be so much tilt like the map

Position (1.)

![image](https://user-images.githubusercontent.com/1283445/180370785-726e809e-deee-496f-b4c3-75107dee1578.png)

Tilt (2.)

![image](https://user-images.githubusercontent.com/1283445/180371070-157c84d6-af13-4229-8a45-5127b056df9a.png)